### DISCUSSION TITLE
Where available and how to use openShare

### DISCUSSION THREAD

**Discussion by an-vadim-an** [Telegram-Mini-Apps/tma.js#434] [General]
I wanted to use openShare to test sending a message from Mini App to other chat rooms. I used the @telegram-apps/sdk library . Is openShare available in it or do I need to use another library?

mainButton.on('click', async () => {
      try {
        await miniApp.openShare({
          text: 'Test',
        });
        console.log('The chat selection window is open to send a message.');
      } catch (error) {
        console.error('Error when opening the chat selection window:', error);
      }
    });

**Comment by heyqbnk** [comment]
What is openShare? Where did you find this method?

**Reply by an-vadim-an** [reply]
I don't remember where exactly I read about it from the forums. But I understand it doesn't exist at all. 
What can I use so that when I click on the main button I can send a message to Telegram chats, like forwarding!

**Reply by heyqbnk** [reply]
> I don't remember where exactly I read about it from the forums. But I understand it doesn't exist at all. What can I use so that when I click on the main button I can send a message to Telegram chats, like forwarding!

Maybe this one:
https://docs.telegram-mini-apps.com/packages/telegram-apps-sdk/components/utils#shareurl

**Reply by an-vadim-an** [reply]
Thank you so much! This is exactly what I needed!
